src/content/cloud/custom-domains.mdx

```diff
@@ -22,13 +22,94 @@ Applications that use custom domains will automatically get issued a valid certi
 
 ## Add a Custom Domain Name to your Application.
 
-Perform the following tasks to register your custom domain in your Okteto Cloud account:
+Perform the following tasks to use your custom domain in an Okteto Namespace:
+
+<Tabs
+  defaultValue="cloud"
+  values={[
+    { label: 'Okteto Cloud', value: 'cloud', },
+    { label: 'SaaS', value: 'saas', },
+    { label: 'Self-Hosted', value: 'self-hosted', },
+  ]}
+>
+<TabItem value="cloud">
 
 1. Confirm that you own the custom domain name. Need a custom domain name? You can buy one with a domain registration service.
 
-2. [Reach out to support](https://okteto.com/support) to register your custom domain with Okteto Cloud. Please include the domain, your github ID, and the namespace you want to use in your communications.
+2. Update your DNS configuration so your custom domain points to [Okteto DNS target](/docs/cloud/custom-domains/#configure-your-dns).
+
+3. [Reach out to support](https://okteto.com/support) to register your custom domain with Okteto Cloud. Please include the domain, your github ID, and the namespace you want to use in your communications.
+
+</TabItem>
+<TabItem value="saas">
+
+1. Update your DNS configuration so your custom domain points to [Okteto DNS target](/docs/cloud/custom-domains/#configure-your-dns).
```

Comment:
This changes per instance no? Maybe we need to ask them to reach out to support? Or make this visible somewhere in the UI?

Reply:
when we talk specifically about SaaS, we don't use cloud URLs or DNS because they are not going to work.  Maybe we can add a note reminding people that the sample uses Okteto Cloud, and to change it to the URL of their instance? (or something like that?)